kratos/tests/cpp_tests/utilities/test_auxiliar_model_part_utilities.cpp

```diff
@@ -18,14 +18,17 @@
 #include "containers/model.h"
 #include "testing/testing.h"
 #include "includes/model_part.h"
-#include "utilities/auxiliar_model_part_utilities.h"
+#include "geometries/triangle_3d_3.h"
 
 // Utilities
+#include "utilities/auxiliar_model_part_utilities.h"
 #include "utilities/cpp_tests_utilities.h"
 
 namespace Kratos {
 namespace Testing {
 
+typedef Node<3> NodeType;
```

Comment:
I would define this in a no named namespace

Reply:
In a different PR, this PR has waited to long

Reply:
Have you changed this one in another PR?

Reply:
https://github.com/KratosMultiphysics/Kratos/pull/10370 now